### PR TITLE
Fix a issue that generates ExtraWhiteSpace

### PR DESCRIPTION
**Pull request by heshiyou**
The PR proposed to resolve the ExtraWhiteSpace issue, which vim takes as a bad character.

### CODE REVIEW

File: autoload/codi.vim [hunk 1]
```diff
@@ -560,8 +560,8 @@ function! s:codi_handle_done(bufnr, output)
       if l =~ i['prompt']
         " If we have passed the first prompt
         if passed_first
-          " Record what was taken (needs to be at least one character)
-          call add(result, len(taken) ? taken : ' ')
+          " Record what was taken, empty if nothing happends
```
Comment (metakirby5) [inline autoload/codi.vim]: Typo: "happends" -> "happens"

Comment (heshiyou) [inline autoload/codi.vim]: sorry for that.

**Comment by heshiyou** [comment]
@metakirby5 The PR mainly removes the whitespace output by codi when REPL nothing generates.
Because Vim take whitespace in blank line as bad behavior and do warning, like this:

image

**Comment by metakirby5** [comment]
Got it. I think there was some reason I made it one character, but I can't recall why. I'll merge this in for now and undo it if it causes a regression.

**Pull request merged**